inmanta.spec

```diff
@@ -53,9 +53,10 @@ Requires:       python36
 Requires:       python3-devel
 %define __python3 /usr/bin/python3.6
 %else
-BuildRequires:  python3-devel
-Requires:       python3
-Requires:       python3-devel
+BuildRequires:  python36-devel
+Requires:       python36
```

Comment:
On rhel/centos 7.7 and 8 only the python3 package exists. For backwards compat it also provides python36. I would prefer a conditional clause that sets the correct depends. On < 7.7 use python36 otherwise python3

Reply:
Like discussed offline, we'll stick with this approach and add the symlink